tests/configuration/io/test_koswat_surroundings_importer.py

```diff
@@ -51,3 +70,112 @@ def test_import_from_raises_when_traject_loc_shp_file_not_given(self):
 
         # 3. Verify expectations.
         assert _expected_error == str(exc_err.value)
+
+    @pytest.fixture(
+        name="surroundings_section_fom_fixture",
+        params=[_stc.values for _stc in _surroundings_test_cases],
+        ids=[_stc.id for _stc in _surroundings_test_cases],
+    )
+    def _get_surroundings_section_fom_fixture(
+        self, request: pytest.FixtureRequest
+    ) -> Iterator[SurroundingsSectionFom]:
+
+        # Create a copy of the directory in a temporary location
+        _case_name = get_fixturerequest_case_name(request)
+        _surroundings_path = request.param[0]
+        _temp_dir = test_results.joinpath(_case_name, _surroundings_path.stem)
+        if _temp_dir.parent.exists():
+            shutil.rmtree(_temp_dir.parent)
+        shutil.copytree(_surroundings_path, _temp_dir)
+
```

Comment:
Why create a copy if you only read the files?

Reply:
Because the reading of surroundings happens by a collection, which we call (wrongly) database.
In each "database" we have subdirectories, each representing a different traject. Each of these subdirectories contains a collection of different `.csv` surrounding files.
We are here testing only if for each of the trajects the surroundings get imported, if I were to give the parent location of one of the directories subsequently the other trajects would also be copied, therefore by copying the target directory into a temporary one I overcome the issue and focus only in the relevant import.